docs/source/api.rst

```diff
@@ -4,31 +4,33 @@ API Reference
+    +-----------------------------------------+--------------------------------------------------------------+
+    | **Subpackage**                          | **Description**                                              |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.diffeq`                  | Probabilistic solvers for ordinary differential equations.   |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.filtsmooth`              | Bayesian filtering and smoothing.                            |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.kernels`                 | Kernels / covariance functions.                              |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.linalg`                  | Probabilistic numerical linear algebra.                      |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.linops`                  | Finite-dimensional linear operators.                         |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.problems`                | Definitions and collection of problems solved by PN methods. |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.quad`                    | Bayesian quadrature / numerical integration.                 |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.randprocs`               | Random processes representing uncertain functions.           |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.randvars`                | Random variables representing uncertain values.              |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.statespace`              | Probabilistic state space models.                            |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :mod:`~probnum.utils`                   | Utility functions.                                           |
+    +-----------------------------------------+--------------------------------------------------------------+
+    | :class:`~probnum._config.Configuration` | Global configuration options.                                |
```

Comment:
I would suggest to have `probnum.config` here for consistency. I assume you used the class to not break linking, but this breaks the pattern of the modules here linking to the sidebar menu entries

Reply:
Not sure, if I understand you, correctly. Do you mean sth like this?

```
:mod:`probnum.config <~probnum._config.Configuration>`
```

Reply:
Because that doesn't work 😆 This way, the side bar works (as before) but in the table, the link is broken.

Reply:
How about this?
```suggestion
    | :class:`config <~probnum._config.Configuration>` | Global configuration options.                                |
```
That way the link should work, even though the destination is the `Configuration` class. In practice users will want to see that doc page anyway.

Reply:
And this does not work?
```suggestion
    | :mod:`config <~probnum._config` | Global configuration options.                                |
```

Reply:
I assume above the link is broken since `probnum._config.Configuration` is not a module.

Reply:
> And this does not work?

For some reason, it does not. Weirdly, the link in the sidebar does work, the link in the table, does not.

Reply:
Then i would suggest to just rename in the table, i.e. to use:

```suggestion
    | :class:`config <~probnum._config.Configuration>` | Global configuration options.                                |
```

Reply:
For some reason, the renaming breaks the link in the table.

Reply:
Ah, the tilde was the problem. Fixed it